db/database.go

```diff
@@ -416,7 +416,7 @@ func (context *DatabaseContext) GetOIDCProvider(providerName string) (*auth.OIDC
 	if provider, ok := context.OIDCProviders[providerName]; ok {
 		return provider, nil
 	} else {
-		return nil, base.RedactErrorf("No provider found for provider name %q", base.UD(providerName))
+		return nil, base.RedactErrorf("No provider found for provider name %q", base.UD(providerName).Redact())
```

Comment:
Not sure this change is nessesary given the `RedactErrorf` type, but it also doesn't feel like user data. @adamcfraser what do you think about removing redaction here?

Reply:
/cc @adamcfraser

Reply:
I think it should still be MD(), but I agree that the change also shouldn't be necessary.